### PR TITLE
[Docs] Translate dev/js/npm.md to zh finished

### PR DESCRIPTION
**Pull request by r-asou**
Signed-off-by: r-asou <aar@webassembly-cn.com>

### CODE REVIEW

File: docs/book/zh/src/dev/js/npm.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+    "rollup-plugin-node-globals": "^1.4.0",
+    "rollup-plugin-async": "^1.2.0"
+  },
+  "scripts": {
+    "build": "rollup -c rollup.config.js"
+  }
+}
+```
+
+执行下方的NPM命令将 [npm_main.js](https://github.com/second-state/wasmedge-quickjs/blob/main/example_js/simple_common_js_demo/npm_main.js) 示例程序构建为 `dist/npm_main.mjs`。
```
Comment (hydai) [inline docs/book/zh/src/dev/js/npm.md:82]: 中英文間要空格

File: docs/book/zh/src/dev/js/npm.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+
+```bash
+$ npm install
+$ npm run build
+```
+
+下方展示的是如何在WasmEdge CLI中执行上方我们生成的JS文件。
```
Comment (hydai) [inline docs/book/zh/src/dev/js/npm.md:89]: 中英文間要空格

File: docs/book/zh/src/dev/rust.md [hunk 3]
```diff
@@ -1 +1,113 @@
 # Rust
+
+在 Webassembly 生态体系中，Rust 是其中一个一等公民。有关 Webassembly 的所有 WasmEdge 扩展程序都为开发者提供了相应的 Rust API 。在这一章节中，我们会为你展示如何将你的Rust应用程序编译为 wasm 字节码，并在 WasmEdge Runtime 中运行你的程序。
+
+## 前置要求
+
+在学习这节内容前，你需要安装 [Rust](https://www.rust-lang.org/tools/install) 以及 [WasmEdge](../start/install.md)。同时，你也需要在Rust工具链中安装 `wasm32-wasi` 。命令如下所示：
```
Comment (hydai) [inline docs/book/zh/src/dev/rust.md:7]: 中英文間要空格

File: docs/book/zh/src/dev/rust.md [hunk 4]
```diff
@@ -1 +1,113 @@
...
+```bash
+$ wasmedge target/wasm32-wasi/debug/hello.wasm second state
+hello
+second
+state
+```
+
+## 一个简单的函数
+
+[add示例](https://github.com/second-state/wasm-learning/tree/master/cli/add) 是一个可以通过[WasmEdge CLI](../start/cli.md)以reactor模式执行的 Rust library 函数。
```
Comment (hydai) [inline docs/book/zh/src/dev/rust.md:49]: 中英文間要空格

File: docs/book/zh/src/dev/rust.md [hunk 5]
```diff
@@ -1 +1,113 @@
...
+我们会使用 `wasmedge` 以 reactor 模式来运行这个程序. 我们会以命令行参数的形式传入我们要调用的方法名以及对应的入参。
+
+```bash
+$ wasmedge --reactor target/wasm32-wasi/debug/add.wasm add 2 2
+4
+```
+
+### 传入复杂调用参数
+
+当然，在大多数情况下，你们不会通过传入CLI参数来调用函数。相反，你们可能会需要使用 [WasmEdge提供的那些编程语言对应的SDK](../../embed.md) 来调用函数。传入参数，然后就能得到返回结果。下方展示的这些SDK案例就是告诉你们如何传入复杂参数并获取返回值。
```
Comment (hydai) [inline docs/book/zh/src/dev/rust.md:79]: 中英文間要空格

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by alabulei1** [comment]
Thanks for your contribution. This PR stuck here for a long time and behind lots of commits. Since #1567   has finished, so we will close your PR.

**Pull request closed without merging**